source/Grid/Grid.js

```diff
@@ -284,9 +265,9 @@ export default class Grid extends PureComponent {
     estimatedRowSize: 30,
     getScrollbarSize: scrollbarSize,
     noContentRenderer: () => null,
```

Comment:
The problem with repro is probably because in tryflow you operate within the same module.